source/css/sass/_normalize.scss

```diff
@@ -94,6 +94,18 @@ small {
   @include small-spacing();
 }
 
+sub,
```

Comment:
We should keep commits clean and not include other things. (Means there will be a small merge conflict, but it's easy to resolve.)